mmdet3d/datasets/nuscenes_dataset.py

```diff
@@ -211,6 +211,7 @@ def get_data_info(self, index: int) -> Dict[str, Any]:
 
         data = dict(
             token=info["token"],
+            sample_idx=info['token'],
```

Comment:
By this method I solved image_idx = example["sample_idx"]
KeyError: 'sample_idx' problem